src/config.rs

```diff
@@ -599,6 +600,7 @@ impl Default for RaftDbConfig {
             wal_ttl_seconds: 0,
             wal_size_limit: ReadableSize::kb(0),
             max_total_wal_size: ReadableSize::gb(4),
+            max_background_jobs: 6,
```

Comment:
Is 6 is too large for raftdb? Since the total dataset is not large for this db.

Reply:
seem they both use the same background pool?

Reply:
Yes, so i'm wondering if this can make the kv-db's compaction starve for a while.